### PR TITLE
Anytype.Anytype version 0.23.5

### PR DESCRIPTION
**Pull request by larryare**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/45732)

### CODE REVIEW

File: manifests/a/Anytype/Anytype/0.23.5/Anytype.Anytype.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Anytype.Anytype
+PackageVersion: 0.23.5
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://download.anytype.io/?action=download&key=desktop&id=54367950
```
Comment (Trenly) [inline manifests/a/Anytype/Anytype/0.23.5/Anytype.Anytype.installer.yaml:9]:
```suggestion
  InstallerUrl: https://at9412003.fra1.cdn.digitaloceanspaces.com/Anytype%20Setup%200.23.5.exe
```

File: manifests/a/Anytype/Anytype/0.23.5/Anytype.Anytype.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Anytype.Anytype
+PackageVersion: 0.23.5
+PackageLocale: en-US
+Publisher: Anytype
+PackageName: Anytype
+License: Copyright © 2022 Anytype
```
Comment (Trenly) [inline manifests/a/Anytype/Anytype/0.23.5/Anytype.Anytype.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
Copyright: Copyright © 2022 Anytype
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/45732#pullrequestreview-879410369

**Comment by zachcarp** [comment]
@denelon - I can't seem to find an installer for this through traditional means.

**Comment by denelon** [comment]
@larryare is there an official website for Anytype with a reference for an "official" download location?

**Comment by lechacon** [comment]
There is https://download.anytype.io/, but it does not seem to be linked from anywhere on https://anytype.io, which says it is in a closed alpha you need to register for.

**Comment by larryare** [comment]
> @larryare is there an official website for Anytype with a reference for an "official" download location?

What @lechacon said is true, software is closed alpha now but Winget is the first source I search for software in Windows. There's an AUR package for Arch Linux, and in my opinion it would be useful to have it in Winget too.

**Comment by zachcarp** [comment]
I went to the download domain provided by @lechacon and ran a redirect trace: https://download.anytype.io/
image

This matches the URL provided by user.

File: manifests/a/Anytype/Anytype/0.23.5/Anytype.Anytype.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Anytype.Anytype
+PackageVersion: 0.23.5
+PackageLocale: en-US
+Publisher: Anytype
+PackageName: Anytype
+License: Proprietary
+Copyright: Copyright © 2022 Anytype
+ShortDescription: Anytype
```
Comment (OfficialEsco) [inline manifests/a/Anytype/Anytype/0.23.5/Anytype.Anytype.locale.en-US.yaml:11]: Needs a better ShortDescription

Once you're at it you can add PublisherUrl, PackageUrl and Tags

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Pull request closed without merging**